### DISCUSSION TITLE
Force native Charge On Hold feature of macOS?

### DISCUSSION THREAD

**Discussion by mrdc** [mhaeuser/Battery-Toolkit#11] [Ideas]
Hello,

Is it possible to force enable native Charge On Hold feature of macOS?

**Comment by mhaeuser** [comment]
At least not easily. What do you expect this to do except for a different message in the native battery menu?

**Comment by mrdc** [comment]
> At least not easily. What do you expect this to do except for a different message in the native battery menu?

Haven't checked Battery-Toolkit sources yet to see how it works under the hood. So it can use the same system stuff as native battery management :D

**Reply by mhaeuser** [reply]
What “same system stuff”? This is likely controlled by some private framework and it probably has an unstable API. BT uses the SMC keys to turn off charging or AC.

**Reply by mrdc** [reply]
Just a guess :) I haven't checked Battery-Toolkit internals, so I'm totally not aware about an approach it uses.
